dbterd/adapters/targets/mermaid/mermaid_test_relationship.py

```diff
@@ -14,6 +17,29 @@ def run(manifest, catalog, **kwargs):
     return ("output.md", parse(manifest, catalog, **kwargs))
 
 
+def replace_column_name(column_name: str) -> str:
+    return column_name.replace(" ", "-").replace(".", "__")
+
+
+def match_complex_column_type(column_type: str) -> Optional[str]:
+    pattern = r"(\w+)<(\w+\s+\w+(\s*,\s*\w+\s+\w+)*)>"
+    match = re.match(pattern, column_type)
+    if match:
+        return match.group(1)
+    else:
+        return None
+
+
+def replace_column_type(column_type: str) -> str:
+    # Some specific DWHs may have types that cannot be drawn in mermaid, such as `Struct<first_name string, last_name string>`.
+    # These types may be nested and can be very long, so omit them
```

Comment:
Let's use Python Docstring here @syou6162 

PS: I'm using [this vscode extension](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) for quickly generating it ;-)

Reply:
@datnguye I added Docstring in [d5a402f](https://github.com/datnguye/dbterd/pull/85/commits/d5a402f7abb59361f679a1ff854a25a67472165e)!